src/01-functions.js

```diff
@@ -9,7 +9,11 @@
  *  alwaysTrue();
  *  //> true
  */
-function alwaysTrue() {}
+function alwaysTrue() {
+  if (typeof boolean){
```

Comment:
Let's take another look here.  What happens if a pass a `number` to this function?  WiIl it ` return true ` as it should?